thirdparty/README.md

```diff
@@ -606,15 +606,14 @@ instead of `miniz.h` as an external dependency.
 ## thorvg
 
 - Upstream: https://github.com/Samsung/thorvg
-- Version: 0.7.0 (e527f565b770f0a41df821e6618ccaeea94f465e, 2021)
+- Version: 0.7.1 (d53eb2a880002cb770ace1c1ace9c5dfcfc28252, 2022)
 - License: MIT
 
 Files extracted from upstream source:
 
 See `thorvg/update-thorvg.sh` for extraction instructions. Set the version
 number and run the script.
 
-Patches in the `patches` directory should be re-applied after updates.
 
```

Comment:
There's now 3 lines between the sections instead of 2. Please remove another line (as in my diff).

Reply:
Done.